build.sh

```diff
@@ -103,6 +103,22 @@ for i in "${ARGS[@]}"; do
 	esac;
 done
 
+if ! command -v /bin/bear &> /dev/null; then
+	echo "'bear' executable not found. Compilation database will not be built"
+elif [ -z "${DO_NOT_EXEC_BEAR+x}" ]; then
```

Comment:
i think `+x` should not be needed (you're setting it to non-empty string), but this is also ok (just less common so more knowledge of how bash works is needed)